### ISSUE TITLE
Where is the data being stored for whos looted and lootchests that have been set?

### ISSUE THREAD

**Issue by TomLewis** [sachingorkar102/Lootin-plugin#26]
I can only see a config.yml in the plugin directory, and wish I see the data for all loot chests that have been created using the `set` command. 
Im also interested in where the data is saved for players that have already looted a chest!

Because my main use is using: `/lootin set CHEST|BARREL|MINECART` and I cant list existing loot locations! I could set many in an event world and cant remember where or what was in them!

**Comment by sachingorkar102** [comment]
ah, its stored in chests itself, which is in turn stored in world files i assume, which you cant view 😅

**Comment by TomLewis** [comment]
wow that's really cool, I didn't even know that's possible to store that amount of data in a chest!

Few more questisons

1.  I have other staff allowed to use this command, I want to check they are not stocking loot in an event world we are building
2. Im building the world on creative and will be moving it to survival when done, will this data come with it? 

Would it be possible to keep a list of active Chests so we can find them again and have a list command in-game to TP to them! then we can see the chest, unless I already looted it, then im not sure how I would "reset" my use to check the items in it as Op!

Purely organization and management of lots of chests!

**Comment by TomLewis** [comment]
CMI has a blockinfo command, but its not showing me much info, so maybe just a way to look at a chest and it outputs the information too? to tell you how many people have looted it, would be great stats for knowing if a chest is being used!

![image](https://github.com/sachingorkar102/Lootin-plugin/assets/144046/79a4e11a-1509-4a91-b66a-e2cfdb30e59f)

**Comment by sachingorkar102** [comment]
yes a command like `/lootin view playername/DEFAULT coordinates` to view chest contents for "specified player" and "default contents" is on my plan list

**Comment by sachingorkar102** [comment]
> Im building the world on creative and will be moving it to survival when done, will this data come with it?

it should, if you are moving the world files too

**Comment by TomLewis** [comment]
> yes a command like `/lootin view playername/DEFAULT coordinates-to-container` to view chest contents for "specified player" and "default contents" is on my plan list
> 
> in this command you can keep `coordinates-to-container` empty if you are directly looking at it

Sounds good, will still need a way to list all containers that have been created! if I make 100 for an event, theres no way I will be able to easily keep track!

Goal will be to expand to lots of places on my survival warps and spawn for players to loot once.

back in the day I used to use https://dev.bukkit.org/projects/chestrestock and had the same issue!
This actually had a few other cool features you could "use", like re-stocking a worlds chests on command for everyone, restock a chest using redstone signal (if the chest is setup that way, with a flag) and a broadcast when chests are restocked.

A bit like a "bloodmoon" if you have ever played Breath of the wild or Tears of the kingdom, to tell people a worlds loot has been restocked/reset.

But main thing thats missing for me, would be keeping track of all containers that have been created.

It is a bit of a logic puzzle how to do this, no matter the "database" how chests are stored, YAML/MySQL/Sqlite, its a matter of knowing a chest is active, you can log a chest into the database when created, and remove it when the chest is removed/broken, But if a chest gets wiped say by Worldedit or whatever, then it wont be logged as removed! So then you end up with a database of chests that are no longer actually there, or say that database is corrupted lost or deleted by mistake, then you have no way to populating it again with existing chests in a world. 
Im not sure of the way to solve that, other than scanning all chests in an entire world, which would not be optimal, for say a server thats got a 11 year old world haha (Aka ours).

one option that may not be too resource heavy is when chests are interacted with in a whitelist config option for a world, it checks if that chest is indeed a Lootin chest, and checks that it is in the database. But this wouldnt work too well optimized for a world that isnt an event world, aka a survival where many players spam open/close their chests in their vaults.

**Comment by TomLewis** [comment]
Bump
